js/domUpdates.js

```diff
@@ -190,7 +191,7 @@ const domUpdates = {
       if (player.score > 5) {
         popUpHTML += `
           <p>${player.name}, enter a wager between 5 and ${player.score}</p>
-          <input type="number" class="input--wager" data-player="${index}">
+          <input type="number" class="input--final-wager" data-player="${index}">
```

Comment:
Did you change this name for CSS purposes? It might break the event listener. Maybe you can add a second class and select it in the CSS with that instead.